package.json

```diff
@@ -98,6 +97,7 @@
   },
   "devDependencies": {
     "@pagopa/eslint-config": "^1.1.1",
+    "@pagopa/openapi-codegen-ts": "10.0.1",
```

Comment:
10.0.1? 
And sorry, I didn't noticed it was not in devDependencies

Reply:
Yes, yargs update is breaking code generation. Once in devDependencies this is safe for snyk.

Reply:
Interesting, thanks!

Reply:
updated to 10.0.3